test/e2e/main.go

```diff
@@ -59,7 +59,7 @@ func runSuite() error {
 	glog.Info("PASS: ExpectProviderAvailable")
 
 	glog.Info("RUN: ExpectOneClusterObject")
```

Comment:
`s/ExpectOneClusterObject/ExpectNoClusterObject`

Reply:
we don't need the cluster for machine.openshift.io https://github.com/openshift/cluster-api-provider-aws/pull/151/files#diff-1c49fed4c53a7b1e61a888973c253576R54

Reply:
Longer term we should have something more sophisticated so that we don't have to repeat all these string values.